spec/BowlingSpec.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+describe ('The Bowling Game', function() {
+
+  var game;
+
+  beforeEach(function () {
+    game - new BowlingGame();
+  });
+
+  it('can roll a gutter game', function () {
+      rollMany(1, 20);
+      expect(game.score()).toBe(0);
+  });
+
+  it('can roll all ones', function () {
+    rollMany(1, 20);
+    expect(game.score()).toBe(20);
+  });
```

Comment:
These 2 tests have the same conditions (rollMany(1, 20)) but the expected outcomes are different. Should line 12 be "rollMany(0, 20);"?